src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -50,6 +50,21 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
   // verify that the user is not reviewing their own change request
   if (reviewerId === foundCR.submitterId) return res.status(401).json({ message: 'Access Denied' });
 
+  // if Scope CR, make sure that a proposed solution is selected before approving
+  if (
+    foundCR.type === 'ISSUE' ||
+    foundCR.type === 'DEFINITION_CHANGE' ||
+    foundCR.type === 'OTHER'
+  ) {
+    if (!psId) return res.status(400).json({ message: 'No proposed solution selected' });
```

Comment:
it's also worth checking if a proposed solution exists with that ID first because otherwise it will throw an error if it doesn't exist and you try to update it

Reply:
Shoud I also check if the crId matches the one linked with the provided proposed solution?